test/adagrams.test.js

```diff
@@ -120,7 +121,7 @@ describe("Adagrams", () => {
     });
 
     it("returns a score of 0 if given an empty input", () => {
-      throw "Complete test";
+      expectScores({});
```

Comment:
@vfarinango, you aren't using this function correctly. You would need to pass in something like this: 
```suggestion
      expectScores({ '':0 });
```